maxim/logger/mistral/utils.py

```
@@ -0,0 +1,151 @@
+        else:
+            # Fallback for older SDK versions with `.parts`
+            parts = getattr(content, "parts", None)
+            if parts:
+                for part in parts:
+                    if getattr(part, "text", None):
+                        text += part.text
+
+        return text
+
+    @staticmethod
+    def parse_completion(completion: ChatCompletionResponse) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)